routes/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var express = require('express');
+var router = express.Router();
+const mongoose = require('mongoose');
+const User = require('../modules/userdata');
+
+// BCrypt to encrypt passwords
+
+const bcrypt = require('bcrypt');
+const bcryptSalt = 10;
+
+// -- Connect to mongoose
+
+mongoose.Promise = Promise;
+mongoose.connect('mongodb://localhost/cookie-starter', {
+  keepAlive: true,
+  reconnectTries: Number.MAX_VALUE
+});
+
+/* GET home page. */
+
+router.get('/', (req, res, next) => {
+  res.render('auth/sign-up', { title: 'Express' });
```

Comment:
You have two routes rendering the same template. Why not redirect the / to /auth/sign-up? Also, if you are not using the title info on the page, make sure to remove it from the render